configuration/sitemaps.md

```diff
@@ -537,16 +564,16 @@ Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
                                   >=25="orange", >=15="green", 0="white", <15="blue"]
 ```
 
-In the first example, the `Weather` Item is colored blue (label and value) if the `Temperature` is below or equal to 4°C.
+In the first example, the `Weather` Item is blue (label and value) if the `Temperature` is at or below 4°C.
 
-Expressions will be evaluated from left to right.
-The first condition returning `true` will decide on the color.
-Looking at the second example, you will see, that the given five expressions are ordered in the only meaningful combination.
+The order in which expressions are listed is important.
+Note that expressions are evaluated from left to right, and the first matching expression determines the color.
+Looking at the second example, you will see that the order of the expressions is set so that the first match, left-to-right gives the correct result.
+If the order was reversed, color assignment would not work properly.
 
-`item_name` and `operator` are both optional.
-If not provided, the Item name will default to the current Item and operator will default to `==`.
-In the second example, this is shown by leaving out Temperature and by not giving a comparison operator in the expression `0="white"`.
-The following three lines are totally equal and valid:
+In the second example, the effect of omitting the `item_name` and `operator` parameters is shown by omitting `Temperature` and by omitting the comparison operator in the expression `0="white"` (as compared to `==0="white"`).
+
+The following three lines are equivalent, and all of them are valid:
```

Comment:
I wonder if these should be the first example?

Reply:
Actually, I think this group of examples needs to be redone.  Here is what I would propose:

**Examples**
The following three lines are equivalent:

```perl
Text item=Temperature labelcolor=[>0="blue"] valuecolor=[22="green"]
Text item=Temperature labelcolor=[>0="blue"] valuecolor=[==22="green"]
Text item=Temperature labelcolor=[>0="blue"] valuecolor=[Temperature==22="green"]
```

The line below illustrates the importance of operator order:
```perl
Text item=Temperature valuecolor=[Last_Update=="Uninitialized"="gray",
                                  >=25="orange", >=15="green", 0="white", <15="blue"]
```
Note that expressions are evaluated from left to right, and the first matching expression determines the color.
Looking at the line above, you will see that the first match, left-to-right gives the correct result.
If the order was reversed, the color assignment would not work properly.

Note also, the effect of omitting `Temperature` and omitting the comparison operator in the expression `0="white"` (as compared to `==0="white"`).

Reply:
Sounds good.

Reply:
This comment has been addressed.